README.md

```diff
@@ -57,6 +57,39 @@ The available settings are:
 - `dialogOpen`: If the share dialog is open. Boolean, defaults to `false`.
 - `enabled`: If the plugin is enabled. Boolean, defaults to `true`.
 
+## Extending
+
+There are currently two ways, if you want to extend the `DownloadDialog` component.
+
+### With a plugin hook
+
+The plugin has a `PluginHook` that renders another plugin [next to the `CanvasDownloadLinks`][plugin-hook].
+
+### With the rendering of child components
+
+The `DownloadDialog` component renders `children` [next to the `CanvasDownloadLinks`][children].
+
+To extend the plugin in this way, you can proceed as follows:
+
+1. write your own plugin
+2. add this plugin as dependency
+3. add custom `children`:
+   ```
+   const MyCustomDownloadDialog = (props) => (
+     <DownloadDialog {...props}>
+       <MyCustomChildComponent {...props} />
+     </DownloadDialog>
+   );
+   ```
+   **Note:** `DownloadDialog` is imported from this plugin
```

Comment:
Würde ich in den Code packaen dass es klar ist, einfach `import DownloadDialog from '@mdz/mirador-downloaddialog` oder so, dann ist auch gleich klar ob default oder named exports vorliegen

Reply:
Hab ich umgestellt 👍🏻 danke dir!